app/src/main/java/com/arctouch/codechallenge/util/RequestKeyBuilder.java

```diff
@@ -0,0 +1,18 @@
+package com.arctouch.codechallenge.util;
+
+import java.security.SecureRandom;
+
+public class RequestKeyBuilder {
+
+    public String generateRandomKey(int length)
+    {
+        char[] chars = "abcdefghijklmnopqrstuvwxyz0123456789".toCharArray();
+        StringBuilder sb = new StringBuilder(length);
```

Comment:
Why did you use a `StringBuilder`? Wouldn't a normal `String` concatenation with "+" work too?

Reply:
It actually would work as well, but as I know the size of the requested `String` and I'm using `String` concatenation in a loop, then a `StringBuilder` consumes less memory than concatenating the chars with a `String`.